src/main.js

```diff
@@ -31,6 +32,16 @@ function handleUnitSwitchClick(e) {
   updateWeather(weatherData, unitSystem);
 }
 
+async function handleDisplayWeatherBySearch(event) {
+  let searchText = $("#input").value;
+  let splitText = searchText.split(",");
+  splitText = splitText.map((string) => string.trim()).join(",");
```

Comment:
We could modify the input to the cleared version. It can also indicate to the user how should the search query look like.

```suggestion
  splitText = splitText.map((string) => string.trim()).join(",");
  searchText.value = splitText;
```

Reply:
had to modify this a little bit cause searchText.value is string and was throwing me error...I'm also getting a bad request error when user searches without anything as input..is this ok ok, or should i handle this too?

Reply:
Yes. The API call should not happen when there's nothing to search. When value after trimming and splitting by a comma is an empty array it should just abort. We can implement showing some error messages later.